.gitignore

```diff
@@ -4,3 +4,4 @@
 *.coverprofile
 /dist/
 .vscode
+.tool-versions
```

Comment:
If you're using asdf, this one's probably fine to commit, it doesn't harm the repo and gives us replicability locally